.github/scripts/eval_opencompass_config.py

```diff
@@ -632,110 +660,169 @@
 hf_gemma_chat_7b = dict(type=HuggingFaceCausalLM,
                         abbr='gemma-7b-it-pytorch',
                         path='google/gemma-7b-it',
                         tokenizer_path='google/gemma-7b-it',
-                        model_kwargs=dict(device_map='auto',
-                                          trust_remote_code=True),
-                        tokenizer_kwargs=dict(
-                            padding_side='left',
-                            truncation_side='left',
-                            trust_remote_code=True,
-                        ),
+                        model_kwargs=model_kwargs_template,
+                        tokenizer_kwargs=tokenizer_kwargs_template,
                         meta_template=mistral_meta_template,
-                        max_out_len=256,
-                        max_seq_len=2048,
+                        max_out_len=MAX_NEW_TOKENS,
+                        max_seq_len=MAX_SESSION_LEN,
                         batch_size=16,
                         batch_padding=False,
-                        run_cfg=dict(num_gpus=1, num_procs=1),
+                        run_cfg=run_cfg_tp1_template,
                         end_str='end_of_turn')
 
 pt_gemma_chat_7b = dict(type=LmdeployPytorchModel,
                         abbr='gemma-7b-it-pytorch',
                         path='google/gemma-7b-it',
-                        engine_config=dict(session_len=2048,
-                                           max_batch_size=16),
-                        gen_config=dict(top_k=1,
-                                        top_p=0.8,
-                                        temperature=1.0,
-                                        max_new_tokens=256),
-                        max_out_len=256,
-                        max_seq_len=2048,
+                        engine_config=pt_engine_config_template_max_bs_16,
+                        gen_config=gen_config_template,
+                        max_out_len=MAX_NEW_TOKENS,
+                        max_seq_len=MAX_SESSION_LEN,
                         batch_size=16,
                         concurrency=16,
                         meta_template=gemma_meta_template,
-                        run_cfg=dict(num_gpus=1, num_procs=1),
+                        run_cfg=run_cfg_tp1_template,
                         end_str='<end_of_turn>')
+
+# ===== Configs for meta-llama/Meta-Llama-3-8B-Instruct =====
+# config for llama-3-8b-instruct turbomind
+tb_llama_3_8b_instruct = dict(
+    type=TurboMindModel,
+    abbr='llama-3-8b-instruct-turbomind',
+    path='meta-llama/Meta-Llama-3-8B-Instruct',
+    engine_config=tb_engine_config_template_max_bs_128,
+    gen_config=gen_config_template,
+    max_out_len=MAX_NEW_TOKENS,
+    max_seq_len=MAX_SESSION_LEN,
+    batch_size=128,
+    concurrency=128,
+    meta_template=llama3_meta_template,
+    run_cfg=run_cfg_tp1_template,
+    end_str='[INST]')
+
+tb_llama_3_8b_instruct_kvint4 = dict(
```

Comment:
mmlu result is missing as in here
https://github.com/zhulinJulia24/lmdeploy/actions/runs/9012663244/job/24762243845#step:10:839
```
dataset                                 version    metric    mode    llama-3-8b-instruct-turbomind-kvint4
--------------------------------------  ---------  --------  ------  --------------------------------------
--------- 考试 Exam ---------           -          -         -       -
ceval                                   -          -         -       -
agieval                                 -          -         -       -
mmlu                                    -          -         -       -
GaokaoBench                             -          -         -       -
ARC-c                                   -          -         -       -
--------- 语言 Language ---------       -          -         -       -
WiC                                     -          -         -       -
summedits                               -          -         -       -
chid-dev                                -          -         -       -
afqmc-dev                               -          -         -       -
bustm-dev                               -          -         -       -
cluewsc-dev                             -          -         -       -
WSC                                     -          -         -       -
winogrande                              -          -         -       -
flores_100                              -          -         -       -
--------- 知识 Knowledge ---------      -          -         -       -
BoolQ                                   -          -         -       -
commonsense_qa                          -          -         -       -
nq                                      -          -         -       -
triviaqa                                -          -         -       -
--------- 推理 Reasoning ---------      -          -         -       -
cmnli                                   -          -         -       -
ocnli                                   -          -         -       -
ocnli_fc-dev                            -          -         -       -
AX_b                                    -          -         -       -
AX_g                                    -          -         -       -
CB                                      -          -         -       -
RTE                                     -          -         -       -
story_cloze                             -          -         -       -
COPA                                    -          -         -       -
ReCoRD                                  -          -         -       -
hellaswag                               -          -         -       -
piqa                                    -          -         -       -
siqa                                    -          -         -       -
strategyqa                              -          -         -       -
math                                    -          -         -       -
gsm8k                                   1d7fe4     accuracy  gen     75.21
TheoremQA                               -          -         -       -
openai_humaneval                        -          -         -       -
mbpp                                    -          -         -       -
bbh                                     -          -         -       -
--------- 理解 Understanding ---------  -          -         -       -
C3                                      -          -         -       -
CMRC_dev                                -          -         -       -
DRCD_dev                                -          -         -       -
MultiRC                                 -          -         -       -
race-middle                             -          -         -       -
race-high                               -          -         -       -
openbookqa_fact                         -          -         -       -
csl_dev                                 -          -         -       -
lcsts                                   -          -         -       -
Xsum                                    -          -         -       -
eprstmt-dev                             -          -         -       -
lambada                                 -          -         -       -
tnews-dev                               -          -         -       -
```

Reply:
> mmlu result is missing as in here https://github.com/zhulinJulia24/lmdeploy/actions/runs/9012663244/job/24762243845#step:10:839
> 
> ```
> dataset                                 version    metric    mode    llama-3-8b-instruct-turbomind-kvint4
> --------------------------------------  ---------  --------  ------  --------------------------------------
> --------- 考试 Exam ---------           -          -         -       -
> ceval                                   -          -         -       -
> agieval                                 -          -         -       -
> mmlu                                    -          -         -       -
> GaokaoBench                             -          -         -       -
> ARC-c                                   -          -         -       -
> --------- 语言 Language ---------       -          -         -       -
> WiC                                     -          -         -       -
> summedits                               -          -         -       -
> chid-dev                                -          -         -       -
> afqmc-dev                               -          -         -       -
> bustm-dev                               -          -         -       -
> cluewsc-dev                             -          -         -       -
> WSC                                     -          -         -       -
> winogrande                              -          -         -       -
> flores_100                              -          -         -       -
> --------- 知识 Knowledge ---------      -          -         -       -
> BoolQ                                   -          -         -       -
> commonsense_qa                          -          -         -       -
> nq                                      -          -         -       -
> triviaqa                                -          -         -       -
> --------- 推理 Reasoning ---------      -          -         -       -
> cmnli                                   -          -         -       -
> ocnli                                   -          -         -       -
> ocnli_fc-dev                            -          -         -       -
> AX_b                                    -          -         -       -
> AX_g                                    -          -         -       -
> CB                                      -          -         -       -
> RTE                                     -          -         -       -
> story_cloze                             -          -         -       -
> COPA                                    -          -         -       -
> ReCoRD                                  -          -         -       -
> hellaswag                               -          -         -       -
> piqa                                    -          -         -       -
> siqa                                    -          -         -       -
> strategyqa                              -          -         -       -
> math                                    -          -         -       -
> gsm8k                                   1d7fe4     accuracy  gen     75.21
> TheoremQA                               -          -         -       -
> openai_humaneval                        -          -         -       -
> mbpp                                    -          -         -       -
> bbh                                     -          -         -       -
> --------- 理解 Understanding ---------  -          -         -       -
> C3                                      -          -         -       -
> CMRC_dev                                -          -         -       -
> DRCD_dev                                -          -         -       -
> MultiRC                                 -          -         -       -
> race-middle                             -          -         -       -
> race-high                               -          -         -       -
> openbookqa_fact                         -          -         -       -
> csl_dev                                 -          -         -       -
> lcsts                                   -          -         -       -
> Xsum                                    -          -         -       -
> eprstmt-dev                             -          -         -       -
> lambada                                 -          -         -       -
> tnews-dev                               -          -         -       -
> ```

I need to test so kill it. I will rerun all  models tonight when cuda is free.